app/components/tags_input/tags_input.css

```diff
@@ -1,53 +1,144 @@
+/* These styles overwrite lots of Tagify's default styles
+ * in rather hacky ways. Be extra careful when changing
+ * styles and make sure to test your changes manually.
+ */
 @import '@yaireo/tagify/dist/tagify.css';
 
 .TagsInput {
   width: 100%;
 }
 
-.TagsInput tags {
-  --tag-text-color: #fff;
-
-  --tag-remove-btn-color: #fff;
-  --tag-remove-btn-bg--hover: var(--color-pink);
-  --tag-remove-bg: var(--color-pink);
-
-  --tags-border-color: var(--color-border);
-  --tags-focus-border-color: var(--color-focus);
-
+.tagify {
   border-radius: var(--border-radius);
-
   transition: none;
 }
 
-.TagsInput tags:focus-within {
+.tagify:focus-within {
   box-shadow: var(--input-focus-shadow);
+  border-color: var(--color-focus);
+}
+
+.tagify[aria-expanded='true'] {
+  border-bottom-left-radius: 0;
+  border-bottom-right-radius: 0;
+}
+
+.tagify__input {
+  padding-left: calc(0.125 * var(--spacing-unit));
 }
 
-.TagsInput tag {
+.tagify__tag {
+  transform: translateY(-1px);
   margin-bottom: 0;
+
   font-size: var(--font-size-s);
+  border: 1px solid var(--tag-color);
+  background-color: #fff;
+  border-radius: 9999px;
+
+  transition: none !important;
+  overflow: hidden;
 }
 
-.TagsInput-dropdownItem {
-  display: flex;
-  width: 100%;
-  justify-content: space-between;
+.tagify__tag > div {
+  padding: 0.25em 0.75em;
```

Comment:
Instead of hacking the CSS retrospectively, can we maybe configure taggify to produce the desired CSS output? 

CSS variables: https://github.com/yairEO/tagify#css-variables

SCSS variables: https://github.com/yairEO/tagify/blob/master/src/tagify.scss#L9-L24